infrastructure/ami/hcl2-files/variables.pkr.hcl

```diff
@@ -0,0 +1,45 @@
+variable "region" {
+  description = "The AWS region"
+  type        = string
+}
+
+variable "instance_type" {
+  default     = "trn1.2xlarge"
+  description = "EC2 machine type for building AMI"
+  type        = string
+}
+
+variable "source_ami" {
+  default     = "ami-0fbea04d7389bcd4e" # To get latest AMI use: aws ec2 describe-images --region us-east-1 --owners amazon --filters 'Name=name,Values=Deep Learning AMI Neuron PyTorch 1.13 (Ubuntu 20.04) ????????' 'Name=state,Values=available' --query 'reverse(sort_by(Images, &CreationDate))[:1].ImageId' --output text
+  description = "Base Image"
+  type        = string
+}
+
+variable "ssh_username" {
+  default     = "ubuntu"
+  description = "Username to connect to SSH with"
+  type        = string
+}
+
+variable "optimum_version" {
+  description = "Optimum Neuron version to install"
+  type        = string
+}
+
+variable "transformers_version" {
+  default     = "4.36.2"
+  description = "Transformers version to install"
+  type        = string
+}
+
+variable "ami_users" {
+  default     = ["754289655784", "558105141721"]
+  description = "AWS accounts to share AMI with"
```

Comment:
Are we sure we want to share this on a public repository ?

Reply:
Umm, then we add it in secrets somewhere I guess. 
@philschmid wdyt?

Reply:
Yes AWS Account IDs are not sensitive.

Reply:
> While account IDs, like any identifying information, should be used and shared carefully, they are not considered secret, sensitive, or confidential information. 

https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-identifiers.html#:~:text=While%20account%20IDs%2C%20like%20any,Canonical%20user%20ID